.gitignore

```diff
@@ -37,13 +68,9 @@ coverage.xml
 node_modules/
 bower_components/
 .grunt/
+vendor/
+static/vendor/
 src/vendor/
-src/static/fonts/
```

Comment:
[I did it.](https://github.com/cfpb/design-manual/commit/c303bf93a34f3d45023e8a84a5b514eacf64500c) _( 🎶 Do you think I've gone too far?)_

Not sure why. Anyway, I'll fix it by bringing them in from cf-icons.